test/data/legalDeclationHTML.html

```diff
@@ -39,7 +39,7 @@
     <div class="phase-banner-beta">
         <p>
             <strong class="phase-tag">Beta</strong>
-            <span>This is a new service – your <a aria-label="Feedback: this link will open in a new tab. You&rsquo;ll need to return to this tab to continue with your application." href="https://www.smartsurvey.co.uk/?pageurl=/declaration" target="_blank" rel="external noopener">feedback</a> will help us to improve it.</span>
+            <span>This is a new service – your <a aria-label="Feedback: this link will open in a new tab. You&rsquo;ll need to return to this tab to continue with your application." href="https://www.smartsurvey.co.uk/s/Probate_Feedback/?pageurl=/declaration" target="_blank" rel="external noopener">feedback</a> will help us to improve it.</span>
```

Comment:
Shouldn't the feedback href use something like "{{ links.survey | safe }}?pageurl=/declaration" for consistency?